cpp/src/io/json/nested_json.hpp

```diff
@@ -185,6 +185,73 @@ struct device_json_column {
   }
 };
 
+namespace experimental {
+/*
+ * @brief Sparse graph adjacency matrix stored in Compressed Sparse Row (CSR) format.
+ */
+struct csr {
```

Comment:
IMO this abbreviation is very confusing and cryptic. It is better if we call by full name like:
```suggestion
struct compress_sparse_row {
```